packages/cubejs-athena-driver/driver/AthenaDriver.js

```diff
@@ -13,10 +13,11 @@ class AthenaDriver extends BaseDriver {
       credentials: {
         accessKeyId: config.accessKeyId || process.env.CUBEJS_AWS_KEY,
         secretAccessKey: config.secretAccessKey || process.env.CUBEJS_AWS_SECRET,
+        sessionToken: config.sessionToken || undefined
       },
-      region: process.env.CUBEJS_AWS_REGION,
-      S3OutputLocation: process.env.CUBEJS_AWS_S3_OUTPUT_LOCATION,
-      workGroup: process.env.CUBEJS_AWS_ATHENA_WORKGROUP || 'primary',
+      region: config.region || process.env.CUBEJS_AWS_REGION,
```

Comment:
I don't think that we need to change these lines, because below these changes there is a line: `...config` which will override these variables if the user passes config in the constructor.

wdyt?

Reply:
BTW: You can use:


````js
let config = {
  credentials: {
    accessKeyId: 'myaccessid',
    secretAccessKey: 'myaccesskey',
    sessionToken: 'mysessiontoken',
  }
};

let driver = new AthenaDriver(config);
````

Reply:
I tested the workaround and it works, thanks! I didn't know that ...config would overwrite things. So I guess no change is needed.

Also sorry for the late response I didn't have access to a PC until now to test this.